bedrock/firefox/templates/firefox/windows-64-bit.html

```diff
@@ -19,9 +19,12 @@
  {% block content %}
    <div class="mzp-c-hero">
      <div class="hero-content">
+       <img src="{{ static('protocol/img/logos/firefox/firefox.png') }}" width="50" height="50" alt="">
        <p>{{ _('64-bit') }}</p>
        <h1>{{ _('A more secure Firefox.') }}</h1>
-       <img src="{{ static('protocol/img/logos/firefox/firefox.png') }}" width="50" height="50" alt="">
+       <div class="download-firefox">
+         {{ download_firefox(download_location='primary cta') }}
```

Comment:
Nit: this button has a duplicate ID to the one at the bottom of the page. Suggest giving each one something unique?